saleor/graphql/product/types/products.py

```diff
@@ -1142,3 +1186,33 @@ def resolve_url(root: models.ProductImage, info, *, size=None):
     @staticmethod
     def __resolve_reference(root, _info, **_kwargs):
         return graphene.Node.get_node_from_global_id(_info, root.id)
+
+
+@key(fields="id")
+class ProductImage(graphene.ObjectType):
+    id = graphene.ID(required=True, description="The ID of the image.")
+    alt = graphene.String(required=False, description="The alt text of the image.")
```

Comment:
This field was required before, we should keep it as it was (it should return an empty string if it's not set).